package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "telemesh",
+  "version": "1.0.0",
+  "description": "semantic versioning and change log",
+  "main": "index.js",
+  "directories": {
+    "doc": "docs"
+  },
+  "scripts": {
+    "release": "standard-version",
+    "release:minor": "standard-version --release-as minor",
+    "release:patch": "standard-version --release-as patch",
+    "release:major": "standard-version --release-as major"
+  },
+  "dependencies": {
+    "@commitlint/cli": "latest",
+    "@commitlint/config-conventional": "latest",
+    "husky": "latest"
+  },
+  "husky": {
+    "hooks": {
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
+    }
+  },
+  "commitlint": {
+    "extends": [
+      "@commitlint/config-conventional"
+    ]
+  },
+  "repository": {
```

Comment:
Why its using personal repo?
@W3-Anjan

Reply:
package.json was created with npm init on the telemesh repository. That time I took the personal credential during the configuration. Now I have removed it.